packages/components/src/DateTimePickers/DateTime/error-messages.js

```diff
@@ -1,37 +1,37 @@
-import { t } from 'i18next';
+import i18next from 'i18next';
```

Comment:
It deserves a lint comment
```suggestion
// eslint-disable-next-line import/no-named-as-default-member
import i18next from 'i18next';
```